abstract_key_test.go

```diff
@@ -0,0 +1,139 @@
+/*-
+ * Copyright 2018 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jose
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+)
+
+const (
+	JwsPayload = `{"key":"value"}`
+)
+
+var Key = []byte{0x01, 0x02, 0x03, 0x04, 0x05}
+var WrongKey = []byte{0x06, 0x07, 0x08}
+
+var serialized string
+
+func TesSignWithAbstractSigner(t *testing.T) {
```

Comment:
nit: s/Tes/Test/

Reply:
@alokmenghrajani 

Dude, I spent like 30 min trying to figure out why I could not get this test to run in GoLand. Nice catch.  Fixed in [this commit.](https://github.com/dmercer-google/go-jose/commit/76e8c531fc7f7d186eb92564786befd698e45d5f)